editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy-canvas.spec.tsx

```diff
@@ -167,7 +167,7 @@ function reparentElement(
   return finalEditor
 }
 
-describe('Absolute Reparent Strategy', () => {
+xdescribe('Absolute Reparent Strategy', () => {
```

Comment:
Will these tests be re-enabled subsequently?

Reply:
They either need to go in the bin or be replaced with browser tests. One of those two will be happening in a follow up PR

Reply:
I already deleted this test, because a same was implemented as a browser test already